.changeset/gold-shirts-rescue.md

```diff
@@ -0,0 +1,14 @@
+---
+'@primer/react-brand': minor
+---
+
+Remove `small` size variant to `Label` component. The default `medium` size should be used instead.
+
+⚠️ This is a breaking change
+
+To restore the visual appearance, use the following:
```

Comment:
I don't think they have a way to store visual appearance to small, do they? Shall we remove this bit?